db/migrate/20210728013735_create_users.rb

```diff
@@ -4,6 +4,7 @@ def change
       t.string :uid
       t.string :name
       t.string :email
+      t.string :password_digest
```

Comment:
Remove Password digest

Reply:
just deleted and pushed up let me know if that works.